os/arch/arm/src/amebasmart/amebasmart_i2s.c

```diff
@@ -557,6 +558,8 @@ static void i2s_tx_schedule(struct amebasmart_i2s_s *priv, int result)
 
 		/* Start next transfer */
 		amebasmart_i2s_tx(priv, bfcontainer);
+	} else {
+		ameba_i2s_pause(priv->i2s_object);
```

Comment:
To avoid unexpected case by DMA Timeout, I suggest below
```
		if ((priv->apb_tx->nbytes - priv->apb_tx->curbyte) <= 0)
			ameba_i2s_pause(priv->i2s_object);
		}
```